src/components/flat-table/flat-table-row/flat-table-row.spec.js

```diff
@@ -675,39 +779,12 @@ describe("FlatTableRow", () => {
 
       assertStyleMatch(
         {
-          borderLeft: "2px solid var(--colorsUtilityMajor100)",
+          borderRight: "2px solid var(--colorsUtilityMajor100)",
         },
         wrapper,
-        { modifier: `:focus ${StyledFlatTableRowHeader}` }
+        { modifier: `${StyledFlatTableRowHeader}:nth-child(4)` }
```

Comment:
I'm sure there's nothing wrong with this but I'm a bit puzzled why a test for "applies the correct focus styling" has had the `:focus` modifier removed in the assertion. Or why we need to specify which child it is when there's only one `FlatTableRowHeader` child in this row.

Reply:
Good point I'll update the test description